dom.bs

```diff
@@ -675,6 +680,12 @@ The <dfn attribute for=Event><code>bubbles</code></dfn> and
 <dfn attribute for=Event><code>cancelable</code></dfn> attributes
 must return the values they were initialized to.
 
+<p>The <dfn attribute for=Event><code>returnValue</code></dfn> attribute's getter must return false
+if the <a>canceled flag</a> is set, and true otherwise.
+
+<p>The {{Event/returnValue}} attribute's setter must, if the given value is false, set the <a>canceled flag</a>
+if the {{Event/cancelable}} attribute value is true and the <a>in passive listener flag</a> is unset.
```

Comment:
**context object's** in passive listener flag

Reply:
Also "the context object's {{Event/cancelable}} ..."

Reply:
Although maybe it would be nicer to refactor this and make it share an algorithm with preventDefault(). I can do that before landing I suppose.